contracts/src/pre-deploy/ERC1967Proxy.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Don't copy-past this. Directly import from openzepppelin like this: https://github.com/SpecularL2/specular/pull/99/files#diff-1a8c10fea46dab63836b8dc63381912fd83e15ec4041a2d5bcb136db51e1d933R4

Reply:
agree, I changed this file to just include the import statement. this forces `ERC1967Proxy.sol` to be compiled.